src/akushon/config/utils/config.cpp

```diff
@@ -42,20 +43,30 @@ std::string Config::get_config() const
   for (const auto & action_file : std::filesystem::directory_iterator(path)) {
     std::string file_name = action_file.path();
 
-    try {
-      std::string action_name = "";
-      for (auto i = path.size(); i < file_name.size() - 5; i++) {
-        action_name += file_name[i];
-      }
-      std::cout << action_name << " | ";
+    std::string action_name = "";
+    for (auto i = path.size(); i < file_name.size() - 5; i++) {
```

Comment:
Prefer prefix increment over postfix increment:
```suggestion
    for (auto i = path.size(); i < file_name.size() - 5; ++i) {
```